downstairs/src/repair.rs

```diff
@@ -240,6 +253,39 @@ fn extent_file_list(
 
     Ok(files)
 }
+/// Return the RegionDefinition describing our region.
+#[endpoint {
+    method = GET,
+    path = "/region-info",
+}]
+async fn get_region_info(
+    rqctx: RequestContext<Arc<FileServerContext>>,
+) -> Result<HttpResponseOk<crucible_common::RegionDefinition>, HttpError> {
+    let region_info = rqctx.context().region_dir.clone();
+    let cp = region::config_path(region_info);
+
+    let def: crucible_common::RegionDefinition = match read_json(&cp) {
```

Comment:
Could you instead pass the region definition into the `FileServerContext` at construction?  We're not worried about it changing on-disk, since that would mean we have bigger problems.

Reply:
Yeah, that's a much better idea.
Done in [7bef555](https://github.com/oxidecomputer/crucible/commit/7bef55560bd1a345855ca77b6c296616ee66e0fe)